test.py

```diff
@@ -27,13 +27,30 @@ def print_and_wait(wait_text):
 
 # char_age = input('What is your character\'s age?\n')
 # print ("You are " + char_age + " years old.")
-n = int(input('give me a number, 1-4\n'))
-if n == 1:
-    print('fart')
-elif n == 2:
-    print('wet fart')
-elif n == 3:
-    print('shard')
-elif n == 4:
-    print('poo')
-else: print('you moron!')
\ No newline at end of file
+# n = int(input('give me a number, 1-4\n'))

```

Comment:
What are all these changes? You didn't make any mention of this...

Reply:
Made those lines into comments because I didn't need to test them anymore.